docs/build/message-history.md

```diff
@@ -0,0 +1,136 @@
+---
+sidebar_label: Message history
+sidebar_position: 5.1
+description: Learn how to enable message history for group chat and why this feature is important to providing a seamless group chat user experience.
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Enable message history for group chat with XMTP
+
+Each app installation generates unique group chat keys. This means that group chat messages sent from one app installation can only be decrypted by that same app installation.
+
+To enable users to access their group chat messages using all of their app installations, enable the message history feature. Enabling message history is necessary for a seamless multi-app installation experience.
+
+To learn more about the user experience without the message history feature, see [⚠️ Important: Manage actions that make a local database inaccessible](/docs/build/group-chat/#️-important-manage-actions-that-make-a-local-database-inaccessible).
+
+## Run a message history server
+
+To enable the message history feature, you must run your own [**xmtp-message-history-server**](https://github.com/xmtp/xmtp-message-history-server). When you do, it will provide a message history sync URL to use in the next step.
+
+## Set message history sync URL
+
+When your app creates a new client, specify the location where the client should sync message history.
```

Comment:
Whether you want to use the other message history features or not we recommend atleast setting this url. This allows other apps to pull in message history in the future. The easiest way would be to use our history sync url.